impl/src/main/java/io/kroxylicious/testing/kafka/testcontainers/TestcontainersKafkaCluster.java

```diff
@@ -182,8 +182,10 @@ private EndpointPair buildExposedEndpoint(int brokerId, int internalPort, List<S
     }
 
     private void setDefaultKafkaImage(String kafkaVersion) {
-        DEFAULT_KAFKA_IMAGE = DockerImageName.parse(QUAY_KAFKA_IMAGE_REPO + ":" + kafkaVersion + "-snapshot");
-        DEFAULT_ZOOKEEPER_IMAGE = DockerImageName.parse(QUAY_ZOOKEEPER_IMAGE_REPO + ":" + kafkaVersion + "-snapshot");
+        String kafkaVersionTag = (kafkaVersion.equals("latest")) ? kafkaVersion : "latest-kafka-" + kafkaVersion;
```

Comment:
Make null safe. ..  how about (kafkaVersion == null || kafkaVersion.equals("latest"))

Reply:
By default is defined as `latest` in the `KafkaClusterConfig` class:
```
@Builder.Default
    private String kafkaVersion = "latest";
```
But I can do it if you agree it is a simple check